src/hotspot/share/gc/z/zHeuristics.cpp

```diff
@@ -44,18 +45,22 @@ void ZHeuristics::set_medium_page_size() {
 
   if (size > ZPageSizeSmall) {
     // Enable medium pages
-    ZPageSizeMedium             = size;
-    ZPageSizeMediumShift        = (size_t)log2i_exact(ZPageSizeMedium);
-    ZObjectSizeLimitMedium      = ZPageSizeMedium / 8;
-    ZObjectAlignmentMediumShift = (int)ZPageSizeMediumShift - 13;
+    ZPageSizeMediumMax          = size;
+    ZPageSizeMediumShift        = log2i_exact(ZPageSizeMediumMax);
```

Comment:
Is this an indication that `ZPageSizeMediumShift`should be named `ZPageSizeMediumMaxShift`